blog/_posts/2018-08-08-one-point-zero.md

```diff
@@ -0,0 +1,167 @@
+
+* Scope rules have been simplified. Constructs that introduce local scopes now do so
+  consistently, regardless of whether a global binding for a name already exists or not.
+  This eliminates the “soft/hard scope” distinction that previously existed and means that
+  now Julia can always statically determine whether variables are local or global.
+
+* The language itself is significantly leaner, with many components split out into “standard
+  library” packages that ship with Julia but aren’t part of the “base” language. If you need
+  them, they’re an import away (no installation required) but they’re no longer forced on
+  you. In the future, this will also allow standard libraries to be versioned and upgraded
+  independently of Julia itself, allowing them to evolve and improve at a faster rate.
+
+* We’ve done a thorough review of all of Julia’s APIs to improve consistency and usability.
+  Many obscure legacy names and inefficient programming patterns have been renamed or
+  refactored to more elegantly match Julia's capabilities. This has prompted changes to make
+  working with collections more consistent and coherent, to ensure that argument ordering
+  follows a consistent standard throughout the language, and to incorporate (the now faster)
+  keyword arguments into APIs where appropriate.
+
+* A number of new external packages are being built specifically around the new capabilities 
+  of Julia 1.0. For example:
+    * The data processing and manipulation ecosystem is being revamped to take advantage of
+      the new missingness support.
+    * [Cassette.jl](https://github.com/jrevels/Cassette.jl) provides a powerful mechanism to
+      inject code-transformation passes into Julia’s compiler, enabling post-hoc analysis
+      and extension of existing code. Beyond instrumentation for programmers like profiling
+      and debugging, this can even implement automatic differentiation for machine learning
+      tasks.
+    * Heterogeneous architecture support has been greatly improved and is further decoupled
+      from the internals of the Julia compiler. Intel KNLs just work in Julia. Nvidia GPUs
+      are programmed using the [CUDANative.jl](https://github.com/JuliaGPU/CUDAnative.jl)
+      package, and a port to Google TPUs is in the works.
+
+There are countless other improvements, both large and small. For a complete list of
+changes, see the [0.7 NEWS file](https://docs.julialang.org/en/release-0.7/NEWS/). In our
+original [“Why We Created Julia” blog
+post](https://julialang.org/blog/2012/02/why-we-created-julia) in 2012, we wrote
+
+> It’s not complete, but it’s time for a 1.0 release—the language we’ve created is called
+> [Julia](https://julialang.org).
+
+We may have jumped the gun a bit with mentioning an impending 1.0 release, but the time has
+finally arrived and it is a heck of a release. We are truly proud of what’s been
+accomplished by the thousands of people who have contributed in so many ways to this truly
```

Comment:
two "truly" a bit close to each other, or was it on purpose?

Reply:
No, it was an accident... better word choices for either one appreciated.